src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2661,15 +2661,13 @@ public void TestUsingVarSpecialCase1()
             Assert.Equal(SyntaxKind.LocalDeclarationStatement, statement.Kind());
             Assert.Equal(text, statement.ToString());
             Assert.Equal(0, statement.Errors().Length);
```

Comment:
This file just has unnecessary white space changes. Should revert.

Reply:
Reverted.